### DISCUSSION TITLE
I Can't Open JupyterHub for HW

### DISCUSSION THREAD

**Discussion by alexgoosemann** [cooperrc/advanced-dynamics#18] [General]
I can't open up JupyterHub to complete the homework. I get 403: Forbidden error. WHat do I do. i cleared my cache and everything.
![Screen Shot 2024-01-22 at 3 28 35 PM](https://github.com/cooperrc/advanced-dynamics/assets/157183387/253f3f1c-380d-4cea-9ab8-103b699bff01)

**Comment by cooperrc** [comment]
Ahh, if you're getting this `403 error` I need to add you to the JupyterHub server. 

Send me an email. thanks!
